Source/UserManagement/Web.Angular/src/app/addUser/userForm.component.html

```diff
@@ -0,0 +1,47 @@
+<form [formGroup]='userForm' class="register-staff-form" novalidate>
+    <div class="flex-row">
+        <label for="first_name">First name</label>
+        <input id="first_name" type="text" formControlName='firstName' placeholder="First name" ngDefaultControl>
+    </div>
+    <div class="flex-row">
+        <label>Last name</label>
+        <input id="last_name" type="text" formControlName='lastName' placeholder="Last name" ngDefaultControl>
+        
+    </div>
+    <div class="flex-row">
+        <label>Age</label>
+        <input id="age" type="number" formControlName='age' placeholder="Age" ngDefaultControl>
+    </div>
+    <div class="flex-row">
+        <label for="sex">Sex</label>
+        <select id="sex" class="full-width input-field" formControlName='sex' [(ngModel)]="selectedSex" placeholder="Sex">
+            <option *ngFor="let sex of sexOptions" [value]="sex.value">
+                {{ sex.viewValue }}
+            </option>
+        </select>
+    </div>
+    <div class="flex-row">            
+        <label>Preferred language</label>
+        <select class="full-width input-field" formControlName='preferredLanguage' [(ngModel)]="selectedLanguage" placeholder="Preferred Language">
+            <option *ngFor="let language of languageOptions" [value]="language.value">
+                {{ language.viewValue }}
+            </option>
+        </select>
+    </div>
+    <div class="flex-row">            
+        <label>GPS location</label>
+        <label for="lat">lat</label>
+        <input id="lat"/>
+        <label for="long">long</label>
+        <input/>
+    </div>
+    <div class="flex-row">            
+        <label for="mobile_number">Mobile phone number(s)</label>
+        <input id="mobile_number" type="tel" formControlName='mobilePhoneNumber' placeholder="Telephone">
+    </div>
+    <div class="flex-row">            
+        <label for="email">Email address</label>
+        <input id="email" type="email" formControlName='email' placeholder="E-mail Address" ngDefaultControl>
```

Comment:
See https://angular.io/api/forms/EmailValidator, then you can skip the regex as we are already using angular/forms for stuff  like this

Reply:
This is not part of my change. As this PR was created before https://github.com/IFRCGo/cbs/pull/380 was merged, it looks like there are more changes (this included)

Reply:
Validation need to be fixed later anyway... I will merge this now